plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintCheck.kt

```diff
@@ -43,6 +44,10 @@ open class KtlintCheck @Inject constructor(objectFactory: ObjectFactory) : Defau
         }
     }
 
+    @get:SkipWhenEmpty
```

Comment:
Uhhhh. I don't think we want to skip the task when there isn't a `.editorconfig` file.

Reply:
:scream: Thank you for pointing this out!